script/service/users-back.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @author Bogdan Kovalev
+ */
+
+angular.module('users-back', [])
+    .factory('Users', function () {
+
+        function Users() {
+            var currentUser = "Bogdan";
```

Comment:
Bad idea when backend decides who is current user. User can be set by ng-init in front and then it should be passed to backend if needed.